### ISSUE TITLE
[Start-Here] Term "Temperature growth" does not exists

### ISSUE THREAD

**Issue by HLWeil** [nfdi4plants/nfdi4plants.knowledgebase#529]
### Article URL

https://nfdi4plants.github.io/nfdi4plants.knowledgebase/start-here/study/#add-a-factor-building-block

### Bug

Term "Temperature growth", which you should add as a Factor does not exists.

Maybe use "temperature day" instead?

**Comment by Freymaurer** [comment]
@StellaEggels what would you recommend for the docs?

**Comment by StellaEggels** [comment]
Yes I would also use temperature day (DPBO_0000007), or alternatively temperature of air (ENVO_09200001), I think both work equally well.

**Comment by Freymaurer** [comment]
@HLWeil i just noticed in the text, that it is intended that this term does not exist? 

> 1. In the *Building Blocks* widget, select *Factor*, search for `growth temperature`, if the term does not appear in the list, you can think of other names that match the *Factor*, **in this case we will simply not select anything.** 

@Brilator

**Comment by HLWeil** [comment]
Reads a bit weirdly. If this is intended to be non-existing, I would expect a more explicit explanation. E.g. "..not all concepts are covered by existing ontologies. If you can't find a semantically matching term do xy...
"

**Comment by Brilator** [comment]
I guess somebody just took a short cut: it was missing unintentionally, so the text was adapted to fit somehow...
